keycache/keycache.go

```diff
@@ -65,14 +65,20 @@ func (usage Usage) matchesLabel(labels []string) bool {
 		return true
 	}
 
-	for _, validLabel := range usage.Labels {
-		for _, label := range labels {
-			if label == validLabel {
-				return true
+	//Given an asset usage label must be a superset
```

Comment:
Though this code predates some of our best practices (and the comment does match the style of other comments), it should follow standard Go practices now; e.g.

```
// Given an asset, usage label must be a superset.
```